WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -109,10 +110,11 @@ class PostListFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        val onlyUser = requireNotNull(arguments).getBoolean(EXTRA_POST_LIST_ONLY_USER)
```

Comment:
Similar to my FluxC comments, I am not a big fan of using a `boolean` for a filter. It makes it harder to extend on and a `boolean` is not a meaningful value itself, so we end up having to make up for it with naming which only gets us so far.

I'll leave the decision of how to improve this, but I think we should at least improve the naming as `onlyUser` doesn't carry any meaning for me if I am looking at it outside of this author filter PR.

Reply:
Resolved in 9d1ad99596075800ffeb767c723b6354065e2cb6